universal.go

```diff
@@ -8,25 +8,26 @@ type UniversalOptions struct {
 	// Either a single address or a seed list of host:port addresses
 	// of cluster/sentinel nodes.
 	Addrs []string
-
+	
```

Comment:
Can you please revert all whitespace-only changes like this?

Reply:
Will do. My editor inserts blank spaces at EOF as default.